courses/free-courses-pt_BR.md

```diff
@@ -113,8 +113,8 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
 * [Curso de Elixir Alquimia](https://www.youtube.com/playlist?list=PLv3nyCBtlWP8I9rknIrfcJWrO05yEzknD) - Alquimia Stone
+* [Elixir (Linguagem de Programação)](https://www.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9) - ELLYACADEMY
```

Comment:
no ALL CAPS

Reply:
You can change as per your wish as merge to production @eshellman

Reply:
We have 150 other PRs to deal with. ELLYACADEMY is ALL CAPS

Reply:
@eshellman ohk , i understand ive made changes likewise